package.json

```diff
@@ -10,8 +10,7 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server --proxy http://localhost:8000",
-    "test": "ember test",
-    "docs": "ember ember-cli-yuidoc"
```

Comment:
No more yuidoc? (Has this ever been used?)

Reply:
No it hasn't - that's why..